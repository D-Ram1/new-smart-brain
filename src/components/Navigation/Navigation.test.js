import React from 'react';
import { shallow } from 'enzyme';
import Navigation from './Navigation';

it('renders without crashing', () => {
    expect(shallow(<Navigation />)).toMatchSnapshot();
})