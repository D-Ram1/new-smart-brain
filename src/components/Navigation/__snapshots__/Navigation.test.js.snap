// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<nav
  style={
    Object {
      "display": "flex",
      "justifyContent": "flex-end",
    }
  }
>
  <p
    className="f3 link dim black underline pa3 pointer"
    onClick={[Function]}
  >
    Sign In
  </p>
  <p
    className="f3 link dim black underline pa3 pointer"
    onClick={[Function]}
  >
    Register
  </p>
</nav>
`;
